pkg/ansiescape/split.go

```diff
@@ -0,0 +1,89 @@
+package ansiescape
+
+import "bytes"
+
+// dropCR drops a terminal \r from the data.
+func dropCR(data []byte) []byte {
```

Comment:
Aren't you better off using [`bytes.Trim`](http://golang.org/pkg/bytes/#Trim) here?